prelude.links

```diff
@@ -1197,13 +1197,21 @@ fun assertEq (actual, expected) server {
 #### APP SERVER ####
 
 sig addSimpleRouteHandler : (String, (String) ~> Page) ~> ()
-fun addSimpleRouteHandler(path,f) { unsafeAddRoute(path, fun(p, _){f(p)}) }
+fun addSimpleRouteHandler(path,f) { addRoute(path,f) }
 
 sig addRoute : (String, (String) ~> Page) ~> ()
-fun addRoute(path, f) { unsafeAddRoute(path, fun(p, _){f(p)}) }
+fun addRoute(path, f) { addLocatedRouteHandler(path, fun(p, _){f(p)}) }
+
+sig defaultErrorPage : (String, String, Location) ~> Page
```

Comment:
It turns out you can't use `page` sugar in the Prelude... but otherwise that seems fine (if somewhat less in the _spirit_ of Links, I would have thought....)